blablahelp-backend/src/test/java/org/nnn4eu/hfische/blablahelp/blablahelpbackend/security/web/ctrl/PublicCtrlIT.java

```diff
@@ -0,0 +1,98 @@
+package org.nnn4eu.hfische.blablahelp.blablahelpbackend.security.web.ctrl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.account.Account;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.account.AccountService;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.account.ERole;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.config.UrlMapping;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.security.SecurityConfig;
+import org.nnn4eu.hfische.blablahelp.blablahelpbackend.security.web.ctrl.model.LoginResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.httpBasic;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.user;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ComponentScan(basePackageClasses = {SecurityConfig.class})
+@ExtendWith(SpringExtension.class)
+@WebMvcTest(controllers = PublicCtrl.class)
+class PublicCtrlIT {
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    AccountService accountService;
+
+    @MockBean
+    UserDetailsService userDetailsService;
```

Comment:
Warum mockst du den account service? Ich würde in einem Integrations test möglichst wenig mocken.
 Du könntest dir das Repo injecten um dannn vor dem Test, User in die Datenbank zu schreiben...